resources/ard.rb

```diff
@@ -72,5 +73,6 @@
   end
   execute BASE_COMMAND do
     command "#{BASE_COMMAND} -configure #{configure_options.join(' ')}"
+    not_if { ard_already_configured?(configure_options) }
```

Comment:
@jazaval does this guard take into account that the `:configure` action allows the use of `configure_options`?

Reply:
It does - check out https://github.com/Microsoft/macos-cookbook/pull/160/files#diff-ea0d45c94dcf0b84818d11e3a33ed79bR10

Reply:
Answering my own question: yes. Here's why:

```ruby
      return false unless configure_options == ['-allowAccessFor -allUsers', '-access -on', '-privs -all']
```